src/browser/webview/mattermost.js

```diff
@@ -3,16 +3,13 @@
 // See LICENSE.txt for license information.
 'use strict';
 
-import {ipcRenderer, webFrame} from 'electron';
+/* eslint-disable no-magic-numbers */
 
-import EnhancedNotification from '../js/notification';
+import {ipcRenderer, webFrame, remote} from 'electron';
 
 const UNREAD_COUNT_INTERVAL = 1000;
-//eslint-disable-next-line no-magic-numbers
```

Comment:
I might accept this one better, in this particular case can be guessed quite quickly instead of defining multiple constants for hour, minute, etc.

Reply:
Ditto.